lib/dor/was_seed/thumbnail_generator_service.rb

```diff
@@ -49,19 +51,25 @@ def self.resize_jpeg(orig_jpeg_file, resized_jpeg_file)
         thumbnail.jpegsave(resized_jpeg_file) # we need to save it to a different file; cleanup is handled by caller
       end
 
-      # Queries the configured CDXJ API for the provided seed URI
+      # Queries the configured CDXJ API for the provided seed URI and returns
+      # the earliest 200 OK snapshot available.
       #
```

Comment:
Instead of simply testing if the URL is in the index, look for the earliest 200 OK snapshot and return its timestamp. This will allow us to ensure that we are screenshotting the best possible page.